internal/driver/glfw/window_desktop.go

```diff
@@ -555,8 +555,7 @@ func keyCodeToKeyName(code string) fyne.KeyName {
 
 	char := code[0]
 	if char >= 'a' && char <= 'z' {
-		// Our alphabetical keys are all upper case characters.
-		return fyne.KeyName('A' + char - 'a')
+		return fyne.KeyName(char ^ 32) // Simple conversion to uppercase by adding 32.
```

Comment:
This feels like a case where the comment was added because the code became unclear. Can you find a happy medium perhaps? Was the previous code too slow?

Reply:
What do you think would be a happy medium? The old code was likely more than fast enough but this should be faster as it is just a bit flip. Flipping the fifth bit to convert to uppercase is a common way to do it in ASCII (https://stackoverflow.com/questions/25821178/convert-lowercase-to-uppercase-using-ascii).

Reply:
I must admit that I kind of find the new code to be easier to read 😅

Reply:
I think I've made it a bit clearer now :)